kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -329,7 +329,12 @@ public void close(Collection<TopicPartition> partitions) {
   @Override
   public void stop() {
     try {
-      postCommitHook.close();
+      flushAll();
```

Comment:
before this call to flushAll i'd add extra log which says we gonna flush due close hook